pycmpfit.pyx

```diff
@@ -112,9 +112,7 @@ cdef class _UserFunc(object):
         #self.shape[0] = <cnp.npy_intp>0
 
     def setFunc(self, func):
-        # deperecated syntax
-        #args = inspect.getargspec(func).args
-        args = inspect.getfullargspec(func).args
```

Comment:
This line needs to stay -- this is the non-deprecated version of this call that I just updated